pkg/docker/docker.go

```diff
@@ -995,7 +995,14 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) error {
 		if signal == syscall.SIGQUIT {
 			buf := make([]byte, 1<<16)
 			runtime.Stack(buf, true)
-			fmt.Printf("%s", buf)
+            f, err := os.Create("/var/log/s2i_docker_stack_trace.log")
+            if err != nil {
+                return
+            }
+            defer f.Close()
+            w := bufio.NewWriter(f)
```

Comment:
I think you forgot to import this package ...

Reply:
Updated.